plenum/persistence/req_id_to_txn.py

```diff
@@ -11,30 +11,36 @@ class ReqIdrToTxn:
     """
 
     def __init__(self, keyValueStorage: KeyValueStorage):
+        self.delimiter = "~"
```

Comment:
It can be a static constant on class level

Reply:
done